build.sbt

```diff
@@ -86,62 +60,60 @@ lazy val kernel =
       name := "trace4cats-kernel",
       libraryDependencies ++= Dependencies.test.map(_ % Test),
       buildInfoKeys := Seq[BuildInfoKey](version, scalaVersion, sbtVersion),
-      buildInfoPackage := "io.janstenpickle.trace4cats.kernel"
+      buildInfoPackage := "io.janstenpickle.trace4cats.kernel",
+      Test / unmanagedSourceDirectories ++= Seq(
```

Comment:
Looks hacky. Can it be replaced with a module dependency?

Reply:
Yeah, it is hacky 😬, I did want to get your opinion on this in general:

`testkit` depends on `kernel`, but `kernel` needs some some of the arbitrary instances to do its own tests now that `core` has been moved. We could try to break up `testkit`, but there is an intractable problem if we want to avoid copy/paste.

`core` contains `RefSpanCompleter`, which is used in the `kernel` tests. I think we could move `RefSpanCompleter` into `kernel` or change testing approach by using a `WriterT` implementation.